benchmarks/polys.py

```diff
@@ -111,12 +193,20 @@ def setup(self, n, impl):
         if impl == 'expr':
             func = self.get_func_expr(*examples.as_expr())
             expected = examples.to_expr(expected)
+
         elif impl == 'dense':
             func = self.get_func_poly(*examples.as_poly())
-            expected = examples.to_poly(expected)
+            if isinstance(expected, list):
+                expected = [examples.to_poly(polynomial) for polynomial in expected] # for the subresultants type methods the output is in form of list.
+            else:
+                expected = examples.to_poly(expected) # for those methods whose output is only a polynomial not a tuple or list.
```

Comment:
I have mentioned this a few times now:

Lines of code should not be this long. It is especially pointless to make the line too long by adding a long comment at the end of the line: the comment can go anywhere so put it on a different line.
```python
          # for those methods whose output is only a polynomial
          expected = examples.to_poly(expected) 
```

You might think that this looks reasonable if you have your editor in fullscreen mode but most people do not do that. Usually I like to view code side by side with other things (e.g. a browser or a document or something) so I do not fullscreen the code window.

Reply:
I apologize for the same mistake again.
I think in that there is no need to wrap the line for the comment. But I should for sure take care of this next time